[2024-07-05T08:36:43.072+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-05T08:36:43.088+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: prod_dag.run_postgre_docker manual__2024-07-05T08:36:09.102039+00:00 [queued]>
[2024-07-05T08:36:43.092+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: prod_dag.run_postgre_docker manual__2024-07-05T08:36:09.102039+00:00 [queued]>
[2024-07-05T08:36:43.092+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-05T08:36:43.104+0000] {taskinstance.py:2330} INFO - Executing <Task(DockerOperator): run_postgre_docker> on 2024-07-05 08:36:09.102039+00:00
[2024-07-05T08:36:43.114+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=533) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-05T08:36:43.116+0000] {standard_task_runner.py:63} INFO - Started process 536 to run task
[2024-07-05T08:36:43.116+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'prod_dag', 'run_postgre_docker', 'manual__2024-07-05T08:36:09.102039+00:00', '--job-id', '316', '--raw', '--subdir', 'DAGS_FOLDER/DAG_prod.py', '--cfg-path', '/tmp/tmpq8n5lkzd']
[2024-07-05T08:36:43.117+0000] {standard_task_runner.py:91} INFO - Job 316: Subtask run_postgre_docker
[2024-07-05T08:36:43.153+0000] {task_command.py:426} INFO - Running <TaskInstance: prod_dag.run_postgre_docker manual__2024-07-05T08:36:09.102039+00:00 [running]> on host 0babcc6ad8d9
[2024-07-05T08:36:43.213+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='prod_dag' AIRFLOW_CTX_TASK_ID='run_postgre_docker' AIRFLOW_CTX_EXECUTION_DATE='2024-07-05T08:36:09.102039+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-07-05T08:36:09.102039+00:00'
[2024-07-05T08:36:43.213+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-05T08:36:43.234+0000] {docker.py:366} INFO - Starting docker container from image bost67dst/postgre_loading
[2024-07-05T08:36:43.822+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-05T08:36:43.830+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=prod_dag, task_id=run_postgre_docker, run_id=manual__2024-07-05T08:36:09.102039+00:00, execution_date=20240705T083609, start_date=20240705T083643, end_date=20240705T083643
[2024-07-05T08:36:43.850+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-05T08:36:43.861+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-05T08:36:43.863+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
